components/canvas/Canvas.module.css

```diff
@@ -0,0 +1,13 @@
+.containermap {
```

Comment:
Use camelcase to make classes better readable
```suggestion
.containerMap {
```